app/lib/admin/backend.dart

```diff
@@ -293,8 +293,9 @@ class AdminBackend {
     List<Version> versionsNames;
     await withRetryTransaction(_db, (tx) async {
       final deletes = <Key>[];
-      final Key packageKey = _db.emptyKey.append(Package, id: packageName);
-      final package = (await tx.lookup([packageKey])).first as Package;
+      final packageKey = _db.emptyKey.append(Package, id: packageName);
+      final package =
+          await tx.lookupValue<Package>(packageKey, orElse: () => null);
```

Comment:
nit. but I added `lookupOrNull` specifically for this scenario :D

(In particular because breaking into multiple lines like this annoyed me.. hehe)